docker/oso-rhel7-zabbix-server/Dockerfile

```diff
@@ -39,4 +39,5 @@ CMD /usr/local/bin/start.sh
 
 # Make the container work more consistently in and out of openshift
 # BE CAREFUL!!! If you change these, you may bloat the image! Use 'docker history' to see the size!
-RUN chmod -R g+rwX /etc/passwd /etc/zabbix /var/log/ /var/run/
+RUN chmod -R g+rwX /etc/passwd /etc/zabbix
+RUN chmod -R 777  /var/log /var/run
```

Comment:
Could we just use this instead?

I hate having files executable that aren't actually binaries.

```
chmod -R g+rwX,o+rwX /etc/passwd /etc/zabbix /var/log /var/run
```

Or if you specifically just want /var/log and /var/run to have 'other' set, then this:

```
chmod -R g+rwX /etc/passwd /etc/zabbix   &&  \ 
chmod -R g+rwX,o+rwX /var/log /var/run
```

I'm also quite sensitive to adding new layers now, so if nothing else, can we combine these two lines into a single run?  :) 

I know I'm picking nits at this point, thanks for working on this.